### PR TITLE
fix(postgresql container): fix container permissions at startup

### LINKED ISSUE #187: [BUG] Chomedriver initializes in different ports, crashing selenium tests on CI
### Describe the bug

Selenium uses the default Chrome Driver port to run the tests, but chromedriver initializes in another port if this one is in use already. Selenium tests must get the correct chromedriver port in order to run.

### To Reproduce

https://github.com/thegraphnetwork/EpiGraphHub/actions/runs/4951358165/jobs/8856724372#step:7:61

### Expected behavior

Rather dynamically gets the service port or find a way of initialize the chromedriver by a harcoded port (2nd option will fail if the port is already in use)

### PR DESCRIPTION
**Pull request by luabida**
fix #187

### CODE REVIEW

File: containers/compose-base.yaml [hunk 1]
```diff
@@ -105,12 +105,12 @@ services:
       - ../.env
     ports:
       - ${POSTGRES_PORT}:${POSTGRES_PORT}
-    user: "${HOST_UID:-1000}:${HOST_GID:-1000}"
```
Comment (xmnlab) [inline containers/compose-base.yaml:108]: any reason for removing this line?

File: containers/compose-base.yaml [hunk 2]
```diff
@@ -105,12 +105,12 @@ services:
...
     environment:
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
       POSTGRES_USER: ${POSTGRES_USER}
       PGDATA: /var/lib/postgresql/data/pgdata
     volumes:
+      - ${POSTGRES_CONFIG_FILE_HOST}:/etc/postgresql/postgresql.conf:ro
```
Comment (xmnlab) [inline containers/compose-base.yaml:113]: maybe you need to add this variable to .env.tpl and to CI:
https://github.com/thegraphnetwork/EpiGraphHub/pull/189/checks#step:11:1180

File: containers/compose-base.yaml (hunk 1 above)
Comment (luabida) [reply containers/compose-base.yaml:108]: postgres user has been properly configured with correct ID and GUI and the container initiates with this user [here](https://github.com/thegraphnetwork/EpiGraphHub/pull/189/files#diff-3fbf821626d0c9d9f6942f5817062a5db43be6434bd83f6906d0492b321f490fR37)

**Comment by luabida** [comment]
```
postgres_1  | 2023-05-16 13:02:11.506 GMT [1] LOG:  unrecognized configuration parameter "stats_temp_directory" in file "/var/lib/postgresql/data/pgdata/postgresql.conf" line 494
postgres_1  | 2023-05-16 13:02:11.506 GMT [1] FATAL:  configuration file "/var/lib/postgresql/data/pgdata/postgresql.conf" contains errors
```

`stats_temp_directory` is deprecated, ref: https://postgresqlco.nf/doc/en/param/stats_temp_directory/

File: containers/compose-base.yaml (hunk 1 above)
Comment (xmnlab) [reply containers/compose-base.yaml:108]: oh, that is right .. sorry .. I missed that it was about the postgres service XD
good catch!

**Review by xmnlab — APPROVED** [review]

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -77,6 +78,8 @@ jobs:
       run: |
         export HOST_UID=$(id -u)
         export HOST_GID=$(id -g)
+        export POSTGRES_HOST_UID=$(id -u)
```
Comment (luabida) [inline .github/workflows/main.yaml:81]: prod has a different user for postgres

**Comment by xmnlab** [comment]
in general, LGTM! just check the error on CI, please

**Comment by luabida** [comment]
Done

**Pull request merged**